dev/TabView/TabView.xaml

```diff
@@ -424,7 +424,7 @@
                                     <VisualState.Setters>
                                         <Setter Target="TabContainer.Background" Value="{ThemeResource TabViewItemHeaderBackgroundSelected}" />
                                         <Setter Target="ContentPresenter.Foreground" Value="{ThemeResource TabViewItemHeaderForegroundPointerOver}" />
```

Comment:
looking at Figma, selected and inactive states have different pointer over and pressed states. Right now, there is the same TabViewItemHeaderForegroundPressed in Pressed and SPressedSelected which doesn't seem correct. Same for TabViewItemHeaderForegroundPointerOver